packages/scheduler/src/components/Scheduler/utils.js

```diff
@@ -10,6 +12,12 @@ export const getResources = calendars => {
   }))
 }
 
+const WHITE_RGB = {r: 255, g: 255, b: 255}
+const getTextColor = hexColor => {
```

Comment:
maybe change signature from `getTextColor(hexColor)` to

`getOptimalTextColor(backgroundHexColor)`

This way it would be clear what this function really returns and what the paramter `hexColor` represents.